src/materials/MeshPhongMaterial.js

```diff
@@ -94,6 +97,9 @@ function MeshPhongMaterial( parameters ) {
 	this.reflectivity = 1;
 	this.refractionRatio = 0.98;
 
+	this.gradientMap = null;
+	this.gradientMapAxisY = false;
```

Comment:
Do you mind removing `gradientMapAxisY` and the related code? I think I would prefer "forcing" the X axis gradient.